stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/ChatUI.kt

```diff
@@ -76,5 +76,5 @@ public object ChatUI {
             mimeTypeIconProviderOverride = value
         }
 
-    public var uiMode: UiMode = UiMode.SYSTEM
+    public var uiMode: UiMode = UiMode.LIGHT
```

Comment:
We shouldn't use `Light` as the default one

Reply:
Ops, this is a mistake.